When running the tests, we had the following warning:
  Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/%{ <-- HERE (\w+)}/ at ./run line 114.
  Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/%{ <-- HERE \?}/ at ./run line 290.
This patch fixes it.

---
 test/run |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)


Index: b/test/run
===================================================================
--- a/test/run
+++ b/test/run
@@ -111,7 +111,7 @@ sub flush_output()
 sub substitute_vars($)
 {
 	my ($line) = @_;
-	$line =~ s[%{(\w+)}][defined $ENV{$1} ? $ENV{$1} : ""]eg;
+	$line =~ s[%\{(\w+)\}][defined $ENV{$1} ? $ENV{$1} : ""]eg;
 	return $line;
 }
 
@@ -287,7 +287,7 @@ while (defined(my $line = <SOURCE>)) {
 	# Parse the next command
 	if ($line =~ s/^\s*\$ ?//) {
 		# Substitute %{?} with the last command's status
-		$line =~ s[%{\?}][$last_status]eg;
+		$line =~ s[%\{\?\}][$last_status]eg;
 
 		chomp($prog = substitute_vars($line));
 		$prog_line = $lineno;
